frontend/src/components/KeyboardShortcutsEducation/KeyboardShortcutsEducation.tsx

```diff
@@ -377,11 +377,11 @@ export const ErrorsKeyboardShortcuts: ShortcutItem[] = [
 	},
 	{
 		description: `Next error`,
-		shortcut: ['j'],
+		shortcut: ['k'],
 	},
 	{
 		description: `Previous error`,
-		shortcut: ['k'],
+		shortcut: ['j'],
```

Comment:
What's the reason for swapping these? `j` for moving down a list and `k` for moving up are pretty standardized. I try them in almost every app I use that presents a list of items I can interact with and there's a lot of muscle memory built up from apps like Gmail, GitHub, Linear, etc. This makes them behave the opposite of what I would expect.

Reply:
Thanks for catching this! Fixed in [4438ce2](https://github.com/highlight/highlight/pull/3578/commits/4438ce23f4c83dd42bbdcc81ccdb15ded550d9fd)

Reply:
Nice! Do we still need to revert the changes to these keyboard shortcut docs?